server/routes/todo.routes.js

```diff
@@ -0,0 +1,23 @@
+const { Router } = require("express");
+const {
+  fetchTodos,
+  getTaskById,
+  addTask,
+  editTaskById,
+  markTaskAsDone,
+  markTaskAsNotDone,
+  removeTodo,
+  clearTodos,
+} = require("../controller/todos.controller");
+const todoRouter = Router();
+
+todoRouter.get("/", fetchTodos);
+todoRouter.get("/:id", getTaskById);
+todoRouter.post("/", addTask);
+todoRouter.put("/task/:id", editTaskById);
+todoRouter.put("/:id", markTaskAsDone);
+todoRouter.patch("/:id", markTaskAsNotDone);
+todoRouter.delete("/", clearTodos);
+todoRouter.delete("/:id", removeTodo);
```

Comment:
Just a tip, you can refactor this for it to look like this:
```suggestion
todoRouter
.route('/')
.get(fetchTodos)
.post(addTask)
.delete(clearTodos);

todoRouter
.route('/:id')
.get(getTaskById);
.put(markTaskAsDone);
.patch(markTaskAsNotDone);
.delete(removeTodo);

todoRouter.put("/task/:id", editTaskById);
```